cfme/physical/physical_server.py

```diff
@@ -293,7 +293,7 @@ def select_entity_rows(self, physical_servers):
         view = navigate_to(self, 'All')
 
         for physical_server in physical_servers:
-            view.entities.get_entity(name=physical_server.name, surf_pages=True).check()
+            view.entities.get_entity(name=physical_server.name, surf_pages=True).ensure_checked()
```

Comment:
added to PRT to cover this. cfme/tests/physical_infrastructure/ui/test_physical_server_list_buttons.py::test_manage_button